### ISSUE TITLE
Add source field to registrations

### ISSUE THREAD

**Issue by arborrow** [arborrow/montserrat#179]
Create source field for participation (registration) on views - add to update, add related methods in RegistrationController - create dropdown list (hardcoded, at least at first). 

I want to look and see how CiviCRM maintains the list of possible sources or if it is just a text field.

**Comment by Djaw** [comment]
@arborrow  requested feature has been added. Let me know how well it works. Please close if everything checks out with testing.

**Comment by arborrow** [comment]
Looks good, I forgot that I had some data from when I brought over the data from ppd in the source field. I added a field ppd_source to the table and moved that data into that field and then cleaned up the source field. I had temporarily removed displaying the source data so as not to confuse folks with the ppd data. I will go ahead and add it back in and then close the issue.

**Comment by arborrow** [comment]
I started to cleanup the blades a bit to simplify things - I believe it will work if we properly create the array to include the name, key combination "N/A" => "N/A". I am tired and will tinker with it tomorrow. Ideally we should declare the registration sources in one place. At times I have used config/polanco.env for this (I think).

**Comment by arborrow** [comment]
OK, I did a little further refactoring, cleaning up. I wanted to avoid repeating the array in the various places as it would mean going through and updating in multiple places if we wanted to add or remove a source. Instead I created the array in the config/polanco.env file and grab that whenever we need it. I am going to go ahead and commit and then cleanup the data that was posted with the array number rather than the actual text.
